rtb/piano_di_progetto_ver0.4.0.typ

```diff
@@ -329,3 +335,50 @@ I casi d'uso sono stati migliorati ed aumentati come pianificato. Tuttavia il gr
 
 L'unica attività che non si è svolta come preventivato è stata l'indagine relativa al _software_ per la gestione di un diagramma di Gantt. L'esigenza era nata in seguito all'utilizzo del sistema integrato di #rifGlossario("GitHub"), il quale però manca di alcune funzionalità, principalmente l'assenza della possibilità di segnalare le dipendenze tra le attività.
 Molte delle soluzioni vagliate, alcune delle quali anche a pagamento, presentavano dei difetti che ci hanno fatto desistere, almeno momentaneamente, dal migrare verso un altro applicativo.
+
+#pagebreak()
+
+=== Sprint 3
+- *Inizio*: 14/12/2024
+- *Fine*: 23/12/2024
+- *Rischi attesi*: #link(<RO-1>)[RO-1], #link(<RO-3>)[RO-3], #link(<RT-1>)[RT-1]
+
+==== Attività dello sprint
+Le attività che il _team_ si è prefissato di portare a termine durante il terzo _sprint_ sono le seguenti:
+- Redazione del verbale esterno ed interno del 13/12/2024.
+
+- Ampliamento del documento Norme di Progetto, in particolare occupandosi delle sezioni:
```

Comment:
Viene segnato "Processi organizzativi" come aggiornamento. In realtà pure quello è un ampliamento. Non è ancora stato messo in main al momento in cui sto scrivendo.